### DISCUSSION TITLE
[Bug]: Error message in the render log

### DISCUSSION THREAD

**Discussion by cesarguvz** [danny-avila/LibreChat#2024] [Troubleshooting] [answered]
### What happened?

Every time I type the api key I get this message in the render log: 

error: There was an uncaught error: Invalid key length

And it keeps asking for the api key

![Captura de pantalla 2024-03-07 183447](https://github.com/danny-avila/LibreChat/assets/97792984/4e2d108d-09e8-4a21-a9f8-2c194309f80b)

to openai api key and anthropic is the same.

### Steps to Reproduce

1. Create a db in mongodb
2. Deploy in render
3. open the web service
4. try to set the api key 

### Relevant log output

```shell
error: There was an uncaught error: Invalid key length
```

### Screenshots

![Captura de pantalla 2024-03-07 223541](https://github.com/danny-avila/LibreChat/assets/97792984/2f2c02d8-ce0a-40bd-a103-9b49dea5d09c)

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by kylerm42** [comment, marked as answer]
I had the same error and tracked it down to this line: https://github.com/danny-avila/LibreChat/blob/main/api/server/utils/crypto.js#L9

So I went looking for the encryption key, and found that it's stored in `CREDS_KEY` in the .env. I used [their generator](https://www.librechat.ai/toolkit/creds_generator) to create a new one, and now no more errors. I realized that although my original key was 64 characters, it was not 32 bytes. Hopefully this helps!

**Reply by danny-avila** [reply]
thanks for sharing!
